sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -13,6 +13,7 @@
 from .blob_service_client import BlobServiceClient
 from .lease import LeaseClient
 from .download import StorageStreamDownloader
+from ._shared_access_signature import generate_account_sas, generate_container_sas, generate_blob_sas
```

Comment:
do we want to rename the file name to public if we want to expose the methods to customers? maybe rename it to shared_access_signature

Reply:
No - we want to keep this file private and only expose the functions via the `__init__.py` files.
As long as they are in `__init__.py` they will be considered public and will be published in the documentation.
This forces people to import them from the package root:
```
from azure.storage.blob import generate_account_sas
```
Rather than:
```
from azure.storage.blob.shared_access_signature import generate_account_sas
```